service/history/queues/action_pending_task_count.go

```diff
@@ -39,12 +39,13 @@ const (
 	clearSliceThrottleDuration = 10 * time.Second
 )
 
+var _ Action = (*actionQueuePendingTask)(nil)
```

Comment:
please add comment to explain what this action is about, what alert is it react to, and how does it mitigate the alert.

Reply:
yeah I will send follow up PRs to comment and add tests for those actions.